src/Features/CSharp/Portable/ConvertIfToSwitch/CSharpConvertIfToSwitchCodeRefactoringProvider.Rewriting.cs

```diff
@@ -60,7 +60,7 @@ private static ExpressionSyntax AsExpressionSyntax(IOperation operation)
             => operation switch
             {
                 IReturnOperation { ReturnedValue: { } value } => (ExpressionSyntax)value.Syntax,
-                IThrowOperation op => ThrowExpression((ExpressionSyntax)op.Exception.Syntax),
+                IThrowOperation { Exception: { } exception } => ThrowExpression((ExpressionSyntax)exception.Syntax),
```

Comment:
This is checked here: https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/ConvertIfToSwitch/AbstractConvertIfToSwitchCodeRefactoringProvider.cs#L113